src/Esfa.Vacancy.Register.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
+VisualStudioVersion = 15.0.27004.2002
```

Comment:
@dev4valtech , @sachpatel  is this VS version difference ok?

Reply:
Are we not using VS 2017 for the Vacancy API? I thought we started out using it?

Reply:
We are having issues running RAA in 2015 and API in 2017, hence didn't upgrade to 2017, however there is no other need to hold back. We should soon upgrade the TeamCity build machines to use new version so RAA can be upgraded as well.

Reply:
Azure SDK upgrade is needed for RAA which can be slightly painful.

Reply:
What's the conclusion here? Revert the numbers?

Reply:
Sounds like we should for now otherwise will have to upgrade RAA + SDK

Reply:
Should we put few tasks to upgrade all our solutions to use VS 2017, latest framework, Azure SDK, plus there are many nuget packages that needs updates as well?

Reply:
yes, I think we should be "fixing forward" 😸